automation/src/test/scala/org/broadinstitute/dsde/test/api/AuthDomainMultiGroupApiSpec.scala

```diff
@@ -295,7 +315,9 @@ class AuthDomainMultiGroupApiSpec extends FreeSpec with Matchers with WorkspaceF
                       val level = getWorkspaceAccessLevel(projectName, workspaceName)(user.makeAuthToken())
                       level should (be("WRITER"))
 
-                      AuthDomainMatcher.checkVisibleAndAccessible(projectName, workspaceName, List(groupOne, groupTwo))(user.makeAuthToken())
+                      eventually {
+                        AuthDomainMatcher.checkVisibleAndAccessible(projectName, workspaceName, List(groupOne, groupTwo))(user.makeAuthToken())
+                      }
```

Comment:
If you're always putting the `AuthDomainMatcher` methods inside an eventually, you might as well change the definitions of those methods and have the `eventually`s in-built. And change the names of them so they're `checkEventuallyVisibleAndAccessible`.

Reply:
👍